frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/controlData/ControlDataHelpers.js

```diff
@@ -718,6 +724,24 @@ function versionGreater(version, x, y) {
   return matches && parseInt(matches[1], 10) >= x && parseInt(matches[2], 10) > y
 }
 
+export const isDeprecatedRHV = (control, _controlData, t) => {
+  if (control.active && control.active.includes('4.13')) {
+    return (
+      <AcmAlert
+        variant="warning"
+        title={t('Deprecated host platform')}
+        message={t(
+          'Red Hat Virtualization is deprecated as a host platform for OpenShift 4.13 and will be removed in the next release.'
+        )}
+        noClose={true}
+        style={{ marginTop: '2em' }}
```

Comment:
```suggestion
        style={{ marginTop: '2em' }}
        isInline
```

I think the drop shadow looks strange in the context of a form. Wonder if we should use inline version?
<img width="1024" alt="image" src="https://github.com/stolostron/console/assets/42188127/a3d2e695-2a72-4583-a438-7b15f5cbdbce">